README.md

```diff
@@ -15,6 +15,7 @@ A curated list of awesome things and projects built by Algerian developpers.
 - [Algerial Logos](https://github.com/mohsenuss91/AlgerianLogos) - Some Algerian official vectorized (.SVG) Logos
 
 ## Applications
+- [Salat Time](https://github.com/Barbarosse/salat_time/) A simple python Web App that returns the five Islamic Prayer Times.
```

Comment:
Hi! thanks for the PR. The list is alphabetically ordered. Can we move this to the right place?

Reply:
Sorry, I didn't notice it.

Reply:
It's fine :) I think you forgot to remove it from the initial position.